infrastructure/deployment/deploy.sh

```diff
@@ -284,14 +312,13 @@ docker_stack_deploy() {
     fi
 
     echo "Downloading $tag"
-
-    until configured_ssh "cd /opt/opencrvs && docker pull $tag"
-    do
-      echo "Server failed to download $tag. Retrying..."
-      sleep 5
-    done
+    echo -e "$tag \n" >> docker_images.txt
   done
-
+  echo "-----"
+  cat docker_images.txt
+  echo "-----"
+  #configured_ssh "cd /opt/opencrvs && docker pull $tag"
```

Comment:
Let's clean up commits and debug messages from PRs

Reply:
sure will do the changes.